css/styles.css

```diff
@@ -134,10 +134,11 @@ header > img {
   margin-top: 1.5rem;
   font-size: 2rem;
   cursor: pointer;
+  transition: none;
```

Comment:
You don't want to put "none" here, because you do want the color to transition when you hover / click.